graf2d/gpadv7/test/attribute.cxx

```diff
@@ -175,3 +175,32 @@ TEST(OptsTest, AttribValue) {
    EXPECT_EQ(value2.Get(), 5);
 
 }
+
+TEST(RAttrTest, EnumValue) {
+
+   enum Style { kNone, kFirst, kSecond, kThird, kForth };
+
+   RAttrValue<Style> value0{kFirst};
```

Comment:
`RAttrValue<enum-type>` isn't used anywhere but in this test. I think it's fine, just saying :-) IIUC, `marker->AttrMarker().SetStyle(RAttrMarker::kStar)` is actually setting a `RAttrValue<int>`.